WcaOnRails/app/models/registration.rb

```diff
@@ -25,11 +26,29 @@ class Registration < ActiveRecord::Base
   end
 
   def pending?
-    accepted_at.nil?
+    accepted_at.nil? && deleted_at.nil?
```

Comment:
Could the tracking be a string column that we append with each action? If we just want a log of events, that seems enough.